src/main.js

```diff
@@ -129,10 +134,10 @@ function makeNewRandomPoster() {
   var i = getRandomIndex(images);
   var t = getRandomIndex(titles);
   var q = getRandomIndex(quotes);
-  var currentPoster = new Poster(images[i], titles[t], quotes[q]);
```

Comment:
Line 132 the use of var here is unnecessary as it's declared above as a global variable